### PR TITLE
Knife Holsters can now hold banana guns.

### PR DESCRIPTION
**Pull request by Kharkov72**
Oooh, banana.
[tested] [oversight]
And yes, the banana gun in the knife holster uses the banana sprite.

:cl:
 * rscadd: Banana Guns now also fit in knife holsters.

### CODE REVIEW

File: code/modules/clothing/accessories/holster.dm [hunk 1]
```diff
@@ -124,7 +124,7 @@
 	holster_verb_name = "Holster (Handgun)"
 
 /obj/item/clothing/accessory/holster/handgun/can_holster(obj/item/weapon/W)
-	if(!isgun(W) && !isbanana(W) && !istype(W, /obj/item/weapon/reagent_containers/food/snacks/grown/bluespacebanana))
+	if(!isgun(W) && !isbanana(W) && !istype(W, /obj/item/weapon/reagent_containers/food/snacks/grown/bluespacebanana) && !istype(W, /obj/item/weapon/gun/projectile/banana))
```
Comment (Shadowmech88) [inline code/modules/clothing/accessories/holster.dm]: Given that banana guns are `/obj/item/weapon/gun/projectile/banana`, wouldn't the `isgun()` check already allow them to be put into holsters?
Did you actually verify that they could not currently be put into holsters?

**Comment by ArchSunder** [comment]
:s clown has a holstered banana, set to arrest

**Comment by Blithering** [comment]
Add regular bananas too

File: code/modules/clothing/accessories/holster.dm (hunk 1 above)
Comment (Kharkov72) [reply code/modules/clothing/accessories/holster.dm]: THAT
is true.
I was reported that they didn't fit in the knife holsters like normal bananas so I assumed.
I'll just remove that.

**Comment by Kharkov72** [comment]
regular bananas are already in, for both handgun and knife holsters.

**Comment by Blithering** [comment]
Oh, then this makes perfect sense to add in that case

**Pull request merged**